source/MaterialXRuntime/RtFileIo.cpp

```diff
@@ -1049,7 +1079,15 @@ namespace
                 ValueElementPtr v = nullptr;
                 if (nodegraphInput.isUniform())
                 {
-                    v = destNodeGraph->addInput(nodegraphInput.getName().str(), nodegraphInput.getType().str());
+                    if (nodegraphInput.isToken())
+                    {
+                        v = destNodeGraph->addToken(nodegraphInput.getName().str());
```

Comment:
Question out of curiousity. If addInput takes a type as a second parameter, shouldn't addToken as well so you don't need two lines to accomplish what could be done on one line?